### DISCUSSION TITLE
Paged Tabs GUI support

### DISCUSSION THREAD

**Discussion by TheSerumDev** [NichtStudioCode/InvUI#63] [Q&A] [answered]
In some cases you want to create a gui where you want to have tabs paged. So what I mean is, you have the control items from the tab ui and you can navigate across multiple pages.

**Comment by NichtStudioCode** [comment, marked as answer]
You can manually set the gui of a control item, then put them in a paged gui.

**Reply by TheSerumDev** [reply]
oh ok, ty i wanna check it out
